### PR TITLE
refactor iteration 0

### PR DESCRIPTION
**Pull request by StevieAmb**
## Branch title

- refactor-iteration0-sa

## Changes made

- Refactored the names of classes and ids to account for developer empathy, and switched all of our boxes to contain display-flex and its associated properties. 

## Anything Refactored

- The titles of classes and ids, and the display properties of elements on the page.

## Anything needed

- Not at the moment.

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,11 +11,11 @@
       <h2 id="page-title">INTENTION TIMER</h2>
     </header>
     <main>
-      <section class="big-box">
+      <section class="activity-presenter">
```
Comment (alexmfritz) [inline index.html:14]: This is a much more accurate name!

File: index.html [hunk 2]
```diff
@@ -36,12 +36,12 @@ <h2 class="activity-title" id="new-activity">New Activity</h2>
...
           </div>
           <div id="accomplish-box">
-            <p class="form-text" id="what-accomplish">What would you like to accomplish during this time?</p>
+            <p class="category-choices" id="what-accomplish">What would you like to accomplish during this time?</p>
             <p><input class="user-input" id="user-accomplish" type="text" name="accomplish"/></p>
           </div>
           <div id="timer-title-box">
-            <p class="form-text" id="minutes">Minutes</p>
-            <p class="form-text" id="seconds">Seconds</p>
+            <p class="category-choices" id="minutes">Minutes</p>
```
Comment (alexmfritz) [inline index.html:43]: These are great class attribute values!

File: index.html [hunk 3]
```diff
@@ -11,11 +11,11 @@
       <h2 id="page-title">INTENTION TIMER</h2>
     </header>
     <main>
-      <section class="big-box">
+      <section class="activity-presenter">
         <h2 class="activity-title" id="new-activity">New Activity</h2>
         <section id="user-activity-input">
-          <p class="form-text" id="category">Select a category:</p>
-          <div class="button-box">
+          <p class="category-choices" id="category">Select a category:</p>
```
Comment (devonwigle) [inline index.html:17]: good choice to remove form confusion since we killed the form

File: styles.css [hunk 4]
```diff
@@ -161,16 +167,18 @@ header {
...
   margin-right: 92px;
   font-size: 17px;
   color: #46424D;
 }
 
 aside {
-  float: left;
-  width: 34%;
-  height: 100vh;
+  display: flex;
```
Comment (alexmfritz) [inline styles.css:177]: Our CSS Flexbox lesson was GREAT!

**Review by devonwigle — APPROVED** [review]
display flex is cool!

**Pull request merged**